src/main/kotlin/io/csbroker/apiserver/common/client/AIServerClient.kt

```diff
@@ -1,6 +1,6 @@
 package io.csbroker.apiserver.common.client
 
-import io.csbroker.apiserver.dto.problem.grade.GradingRequestDto
+import io.csbroker.apiserver.dto.problem.grade.LongProblemGradingRequestToAiServerDto
```

Comment:
🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end